src/tools/compiletest/src/runtest.rs

```diff
@@ -778,6 +778,14 @@ impl<'test> TestCx<'test> {
         script_str.push_str("version\n"); // List CDB (and more) version info in test output
         script_str.push_str(".nvlist\n"); // List loaded `*.natvis` files, bulk of custom MSVC debug
 
+        // If a .js file exists next to the source file being tested, then this is a JavaScript
+        // debugging extension that needs to be loaded.
+        let mut js_extension = self.testpaths.file.clone();
+        js_extension.set_extension("js");
```

Comment:
I think we should require the extension to be `.cdb.js` or something like that -- just to make it more obvious that this is CDB specific and make it easier to search for the relevant code in `compiletest`.

Reply:
Fair enough